3-platform/order/domain/src/main/java/com/_6core/platform/orderdomain/model/Order.java

```diff
@@ -0,0 +1,15 @@
+package com._6core.platform.orderdomain.model;
+
+import java.math.BigInteger;
+import java.util.Set;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table(name = "orders")
+public class Order {
+  @Id private String id;
+  private BigInteger total;
+  private String status;
```

Comment:
let's create on the domain level enum, e.g. OrderStatus. 
For now, you can add only one status: UNDEFINED (or synonym) 
```suggestion
@Enumerated(STRING)
  private OrderStatus status;
```

Reply:
ok, with hibernate-reactive dependency